Yoast/wordpress-seo
# Unique ids for settings snippet editors

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* For testing automation, all input fields should have unique id's. The id for the hidden input fields is passed, with `-snippet-editor` appended.

## Test instructions

This PR can be tested by following these steps:

*NOTE* test together with https://github.com/Yoast/yoast-components/pull/621

* See https://github.com/Yoast/yoast-components/pull/621
* Verify that on in the search-appearance settings, all single fields and snippet editor fields have unique id's that make sense. *NOTE* that you have to look for the div where the class is `Shared__InputContainer`.

_Not testable without making a beta script, I think:_
@dariaknl should be able to use a `click()` function on these id's with puppeteer, and then use `keyPress` to mimic typing. `fillField()` will not work as the divs are not fields.

## Quality assurance


Fixes #10091
